package.json

```
@@ -107,6 +107,7 @@
     "@storybook/manager-webpack5": "6.5.9",
     "@storybook/react": "6.5.9",
     "@storybook/storybook-deployer": "2.8.11",
+    "@storybook/theming": "^6.5.12",
```

suggestion: We version lock all our dependencies [see engineering playbook](https://playbook.truss.dev/docs/practices/appeng/adrs/javascript-dependency-management#version-locking) so please remove the `^` and run `yarn install` again to update the lock file.

```suggestion
    "@storybook/theming": "6.5.12",
```